ganga/GangaCore/Lib/Localhost/Localhost.py

```diff
@@ -184,10 +186,13 @@ def preparejob(self, jobconfig, master_input_sandbox):
         import inspect
 
         fileutils = File( inspect.getsourcefile(GangaCore.Utility.files), subdir=PYTHON_DIR )
+        virtualizationutils = File( inspect.getsourcefile(GangaCore.Utility.Virtualization), subdir=PYTHON_DIR )
 
         sharedfiles = jobconfig.getSharedFiles()
 
-        subjob_input_sandbox = job.createPackedInputSandbox(jobconfig.getSandboxFiles() + [ fileutils ] )
+        subjob_input_sandbox = job.createPackedInputSandbox(jobconfig.getSandboxFiles() + [ fileutils, virtualizationutils] )
```

Comment:
Basically all ok now I think. However we could consider whether we want to be packaging up the virtualisation utility tools with every job or only if a virtualisation is requested. i.e. maybe this line could go inside the 
```python
if virtualization:
    here
```

Reply:
Done had to make minor other changes to make this possible.